apps/site/assets/css/_trip-plan-widget.scss

```diff
@@ -104,9 +116,138 @@
   justify-content: flex-start;
 }
 
-.c-trip-plan-widget__reverse-control {
+.c-trip-plan-widget__reverse-control-vert {
   color: $brand-primary;
   cursor: pointer;
   margin-left: auto;
   user-select: none;
 }
+
+.col-xs-12 {
+  @include trip-planner-widget(true);
+}
+
+.m-tabbed-nav__content-item {
+  @include trip-planner-widget(false);
+  .c-trip-plan-widget {
+    background-color: $gray-bordered-background;
+    border: solid 1px transparent;
+    padding: 0 $base-spacing $base-spacing;
+  }
+  
+  // adjustments for when rendered as a paragraph from the CMS
```

Comment:
Are these styles being duplicated from somewhere? I'm assuming this doesn't affect the home page tab content, so I don't think these are necessary to be added in if they already exist and are affecting the widget in other places

Reply:
Yes a large portion is being duplicated from above. When I just had the homepage specific changes nested within `m-tabbed-nav__content-item` it didn't seem to default to the un nested styles above this section. Could've miscopied something. I'll remove the duplicates and see how it looks.